bd_spot_wrapper/spot_wrapper/spot.py

```diff
@@ -1239,7 +1150,7 @@ def dock(self, dock_id: int = DOCK_ID, home_robot: bool = False) -> None:
     def undock(self):
         # Before undocking from dock, update the global_T_home transform
         # member variable but not don't persist the changes in home.txt
-        self.home_robot(write_to_file=False)
+        # self.home_robot(write_to_file=False)
```

Comment:
no update here?

Reply:
This was actually source of a bug. Thanks for pointing this out. WIll remove it.